pkg/validate/container.go

```diff
@@ -144,6 +160,38 @@ var _ = framework.KubeDescribe("Container", func() {
 			Eventually(getContainerStatus(rc, containerID).ExitCode).Should(Equal(int32(0)))
 		})
 	})
+
+	Context("runtime should support log", func() {
+		var podID string
+		var podConfig *runtimeapi.PodSandboxConfig
+
+		BeforeEach(func() {
+			podID, podConfig = createPodSandboxWithLogDirectory(rc)
+		})
+
+		AfterEach(func() {
+			By("stop PodSandbox")
+			rc.StopPodSandbox(podID)
+			By("delete PodSandbox")
+			rc.RemovePodSandbox(podID)
+		})
+
+		It("runtime should support starting container with log [Conformance]", func() {
+			By("create container with log")
+			logPath, containerID := createLogContainer(rc, ic, "container-with-log-test-", podID, podConfig)
+
+			By("start container with log")
+			// wait container started and check the status.
+			Eventually(testStartContainer(rc, containerID)).Should(BeTrue())
```

Comment:
You may not want to start container again and again.
You should:
```
startcontainer(rc, containerID)
Eventually(verifyContainerStatus(*****), POLL_TIMEOUT, POLL_INTERVAL).Should(BeTrue())
```

And you may not want to use the default 1 second timeout, which is too short.